src/File/Path/Filename/DefaultTrait.php

```diff
@@ -17,6 +17,10 @@ trait DefaultTrait
     public function filename(): string
     {
         $processor = Hash::get($this->settings, 'nameCallback', null);
+        if (is_object($processor) && method_exists($processor, '__invoke')) {
```

Comment:
Why is this additional check necessary? The `is_callable()` check below will already succeed if `$processor` is an object of class with `__invoke()` method https://3v4l.org/qcQVQ.